Apps/W1/Email - SMTP Connector/app/src/SMTPConnectorSenderType.Enum.al

```diff
@@ -0,0 +1,7 @@
+enum 4512 "SMTP Connector Sender Type"
```

Comment:
// ------------------------------------------------------------------------------------------------
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License. See License.txt in the project root for license information.
// ------------------------------------------------------------------------------------------------